pkg/scaffold/v2/controller_suitetest.go

```diff
@@ -130,15 +119,12 @@ var _ = AfterSuite(func() {
 // adding import paths and code setup for new types.
 func (a *ControllerSuiteTest) Update() error {
 
-	a.ResourcePackage, a.GroupDomain = util.GetResourceInfo(a.Resource, a.Repo, a.Domain)
-	if a.Plural == "" {
-		a.Plural = flect.Pluralize(strings.ToLower(a.Resource.Kind))
-	}
+	resourcePackage, _ := util.GetResourceInfo(a.Resource, a.Repo, a.Domain)
 
 	ctrlImportCodeFragment := fmt.Sprintf(`"%s/controllers"
 `, a.Repo)
 	apiImportCodeFragment := fmt.Sprintf(`%s%s "%s/%s"
-`, a.Resource.GroupImportSafe, a.Resource.Version, a.ResourcePackage, a.Resource.Version)
+`, a.Resource.GroupImportSafe, a.Resource.Version, resourcePackage, a.Resource.Version)
 
```

Comment:
The `a.ResourcePackage` is used in this case. 
You are replacing the attribute of the struct for an env var. 
IMO, the previous code is a better approach and follows the same impl of conyttroller.go as well.

Reply:
This has already been merged. We do not need that field in the struct. The struct represents a file object. The file does not require the `ResourcePackage` field for anything. When updating this file we do need it, so we compute it locally there. This was applied to all the files. Every field that is not needed for the file has been removed. `controller.go` does need it for the file (you can make a search for `.ResourcePackage` in the template to see where it is used.